README.rst

```diff
@@ -128,60 +128,31 @@ file or disable package repositories update in config.yaml.
 Using the RPMs
 --------------
 
-We decided that's is a little bit intrusive to install all the
-produced RPMs since some of them may not fit your needs. At the end of
-the process you'll see a new ``result`` directory inside this
-project's root.
+At the end of the process you'll see a new ``result`` directory inside
+this project's root. It contains the built packages repositories under
+``packages`` and yum repository files under ``repository_files`` to
```

Comment:
yum repository files => a yum repository config file

to avoid ambiguity

Also, repository_files => repository_config